packages/cli/src/commands/sandbox/sandbox-devtools/local_storage_manager.ts

```diff
@@ -0,0 +1,886 @@
+import fs from 'fs';
+import path from 'path';
+import { tmpdir } from 'os';
+import { LogLevel, printer } from '@aws-amplify/cli-core';
+import writeFileAtomic from 'write-file-atomic';
+import { BackendResourcesData } from './shared/socket_types.js';
+
+/**
+ * Represents a CloudFormation event
+ */
+export type CloudFormationEvent = {
+  message: string;
+  timestamp: string;
+  resourceStatus?: {
+    resourceType: string;
+    resourceName: string;
+    status: string;
+    timestamp: string;
+    key: string;
+    statusReason?: string;
+    eventId?: string;
+  };
+  isGeneric?: boolean;
```

Comment:
what does generic mean in this context? Can we use a more specific name for this property?